### PR TITLE
Flux overview

### LINKED ISSUE #2769: Add introduction and pointers to the Flux Docs
###### URL for relevant page?
https://docs.influxdata.com/influxdb/v1.8/concepts/crosswalk/

InfluxDB 1.8.0 introduces a new language for working with data, Flux.  For those of you who have worked with InfluxQL and InfluxDB for many years,  Flux is intended to address so many of the outstanding feature requests that we've received since InfluxDB 1.0 was introduced.

Flux is now supported for production usage along side InfluxQL. 

Flux is the primary language for working with data in InfluxDB 2.0 OSS with is currently in beta and InfluxDB 2.0 Cloud which is generally available as a PaaS across a number of Cloud Service Providers.  Introducing Flux into the 1.x line, allows you get started with the concepts, language syntax, and prepare for InfluxDB 2.0 OSS general availability planned for later this year.

### PR DESCRIPTION
**Pull request by sanderson**
Resolves #2769 and #2744 

- Added Flux introduction to crosswalk doc
- Updated Flux vs SQL page with new examples and InfluxQL function links
- Added full list of Flux functions to Flux v0.65

### CODE REVIEW

File: content/influxdb/v1.8/concepts/crosswalk.md [hunk 1]
```diff
@@ -117,7 +111,28 @@ InfluxDB timestamps must be in UNIX epoch (GMT) or formatted as a date-time stri
...
+Flux is the primary language for working with data in [InfluxDB 2.0 OSS](https://v2.docs.influxdata.com/v2.0/get-started)
+(currently in _beta_) and [InfluxDB Cloud 2.0](https://v2.docs.influxdata.com/v2.0/cloud/get-started/),
+a generally available Platform as a Service (PaaS) available across multiple Cloud Service Providers.
+Using Flux with InfluxDB 1.8+ lets you get familiar with Flux concepts and syntax
+and prepare for InfluxDB 2.0 OSS general availability planned for later this year.
+
+### InfluxQL
 
 InfluxQL is an SQL-like query language for interacting with InfluxDB.
 It has been lovingly crafted to feel familiar to those coming from other
```
Comment (timhallinflux) [inline content/influxdb/v1.8/concepts/crosswalk.md:138]: I would add....something like: but it is NOT SQL.  Meaning, if you are a SQL-power user, many of the constructs that you may want to take advantage of are not part of the InfluxQL syntax.  These include UNION, HAVING, Joins, and more.  These capabilities do exist, but you'll need to use Flux to take advantage of these functions and much, much more.

File: content/flux/v0.65/introduction/flux-vs-influxql.md [hunk 2]
```diff
@@ -180,93 +251,135 @@ table2 = from(bucket: "telegraf/autogen")
 cov(x: table1, y: table2, on: ["_time", "_field"])
 ```
 
+### String manipulation and data shaping
+InfluxQL doesn't support any string manipulation when querying data.
```
Comment (kelseiv) [inline content/flux/v0.65/introduction/flux-vs-influxql.md:255]: support string

File: content/flux/v0.65/introduction/flux-vs-influxql.md [hunk 3]
```diff
@@ -106,12 +117,73 @@ from(bucket:"telegraf/autogen")
...
+### Work with multiple data sources
+InfluxQL can only query data stored in InfluxDB.
+Flux can query data from other data sources such as CSV, PostgreSQL, MySQL, Google BigTable, and more.
+Join that data with data in InfluxDB to enrich query results.
+
+- [Flux CSV package](/flux/v0.65/stdlib/csv/)
+- [Flux SQL package](/flux/v0.65/stdlib/sql/)
+- [Flux BigTable package](/flux/v0.65/stdlib/experimental/bigtable/)
+
+<!-- -->
```
Comment (kelseiv) [inline content/flux/v0.65/introduction/flux-vs-influxql.md:154]: is line 154 intentional?

File: content/flux/v0.65/stdlib/all-functions.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+title: Complete list of Flux functions
+description: View the full library of documented Flux functions.
+menu:
+  flux_0_65:
+    name: View all functions
+    parent: Standard library
+weight: 4
+---
+
+This list below contains all documented Flux functions:
```
Comment (kelseiv) [inline content/flux/v0.65/stdlib/all-functions.md:11]: The following list includes

File: content/influxdb/v1.8/concepts/crosswalk.md [hunk 5]
```diff
@@ -117,7 +111,28 @@ InfluxDB timestamps must be in UNIX epoch (GMT) or formatted as a date-time stri
 
 For more detailed descriptions of the InfluxDB terms mentioned in this section see our [Glossary of Terms](/influxdb/v1.8/concepts/glossary/).
 
-## InfluxQL and SQL
+## Query languages
+InfluxDB provides supports multiple query languages:
```
Comment (kelseiv) [inline content/influxdb/v1.8/concepts/crosswalk.md:115]: InfluxDB supports

File: content/influxdb/v1.8/concepts/crosswalk.md [hunk 6]
```diff
@@ -117,7 +111,28 @@ InfluxDB timestamps must be in UNIX epoch (GMT) or formatted as a date-time stri
...
+[Flux](/flux/latest/introduction) is a data scripting language designed for querying, analyzing, and acting on time series data.
+Beginning with **InfluxDB 1.8.0**, Flux is available for production use along side InfluxQL.
+
+For those familiar with [InfluxQL](#influxql-and-sql), Flux is intended to address
+many of the outstanding feature requests that we've received since introducing InfluxDB 1.0.
+For a comparison between Flux and InfluxQL, see [Flux vs InfluxQL](/flux/latest/introduction/flux-vs-influxql).
+
+Flux is the primary language for working with data in [InfluxDB 2.0 OSS](https://v2.docs.influxdata.com/v2.0/get-started)
+(currently in _beta_) and [InfluxDB Cloud 2.0](https://v2.docs.influxdata.com/v2.0/cloud/get-started/),
+a generally available Platform as a Service (PaaS) available across multiple Cloud Service Providers.
```
Comment (kelseiv) [inline content/influxdb/v1.8/concepts/crosswalk.md:131]: a Platform...

File: content/influxdb/v1.8/concepts/crosswalk.md [hunk 7]
```diff
@@ -117,12 +111,36 @@ InfluxDB timestamps must be in UNIX epoch (GMT) or formatted as a date-time stri
...
+### InfluxQL
 
 InfluxQL is an SQL-like query language for interacting with InfluxDB.
-It has been lovingly crafted to feel familiar to those coming from other
+It has been crafted to feel familiar to those coming from other
 SQL or SQL-like environments while also providing features specific
 to storing and analyzing time series data.
+However **InfluxQL is not SQL** and lacks support for more advanced operations
+like `UNION`, `JOIN` and `HAVING` that SQL power-users are accustomed to.
+This functionality is available with [Flux](/flux/latest/introduction).
```
Comment (sanderson) [inline content/influxdb/v1.8/concepts/crosswalk.md:143]: @timhallinflux, I added this statement to address your comment.

File: content/flux/v0.65/introduction/flux-vs-influxql.md (hunk 3 above)
Comment (sanderson) [reply content/flux/v0.65/introduction/flux-vs-influxql.md:154]: Yes, this is a markdown parsing hack. If you include a codeblock after a list, the markdown parser nests the codeblock in the list line item whether its indented or not. Add the comment here forces the markdown parser to close the line item and render the code block separately.

File: content/influxdb/v1.8/concepts/crosswalk.md [hunk 8]
```diff
@@ -117,12 +111,36 @@ InfluxDB timestamps must be in UNIX epoch (GMT) or formatted as a date-time stri
...
+
+For those familiar with [InfluxQL](#influxql-and-sql), Flux is intended to address
+many of the outstanding feature requests that we've received since introducing InfluxDB 1.0.
+For a comparison between Flux and InfluxQL, see [Flux vs InfluxQL](/flux/latest/introduction/flux-vs-influxql).
+
+Flux is the primary language for working with data in [InfluxDB 2.0 OSS](https://v2.docs.influxdata.com/v2.0/get-started)
+(currently in _beta_) and [InfluxDB Cloud 2.0](https://v2.docs.influxdata.com/v2.0/cloud/get-started/),
+a generally available Platform as a Service (PaaS) available across multiple Cloud Service Providers.
+Using Flux with InfluxDB 1.8+ lets you get familiar with Flux concepts and syntax
+and prepare for InfluxDB 2.0 OSS general availability planned for later this year.
```
Comment (timhallinflux) [inline content/influxdb/v1.8/concepts/crosswalk.md:133]: I would like to try and not mention InfluxDB 2.0 OSS GA.....and steer them towards InfluxDB Cloud 2.0 when they need to scale out.   Something along those lines...

File: content/influxdb/v1.8/concepts/crosswalk.md (hunk 7 above)
Comment (timhallinflux) [reply content/influxdb/v1.8/concepts/crosswalk.md:143]: great!

File: content/influxdb/v1.8/concepts/crosswalk.md (hunk 8 above)
Comment (sanderson) [reply content/influxdb/v1.8/concepts/crosswalk.md]: Updated to:

> Using Flux with InfluxDB 1.8+ lets you get familiar with Flux concepts and syntax
and ease the transition to InfluxDB 2.0.

**Pull request merged**